recipes/r-arrow/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = '0.12.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-arrow
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
+  sha256: e93e43343544e344bbc912b89da01d8abf66596f029d26b2b135b102a9e39895
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [win]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
+    - {{ compiler('cxx') }}      # [not win]
+    - {{posix}}pkg-config
+    - {{posix}}make
+  host:
+    - r-base
+    - arrow-cpp {{ version }}
+    - r-r6
+    - r-rcpp >=0.12.18.2
+    - r-assertthat
+    - r-bit64
+    - r-crayon
+    - r-fs
+    - r-glue
+    - r-purrr
+    - r-rlang
+    - r-tibble
+    - r-vctrs >=0.0.0.9000
+    - r-withr
+  run:
+    - r-base
+    - {{ pin_compatible('arrow-cpp') }}
+    - {{native}}gcc-libs         # [win]
+    - r-r6
+    - r-rcpp >=0.12.18.2
+    - r-assertthat
+    - r-bit64
+    - r-crayon
+    - r-fs
+    - r-glue
+    - r-purrr
+    - r-rlang
+    - r-tibble
+    - r-vctrs >=0.0.0.9000
+    - r-withr
+
+test:
+  commands:
+    - $R -e "library('arrow')"           # [not win]
+    - "\"%R%\" -e \"library('arrow')\""  # [win]
+
+about:
+  home: https://github.com/apache/arrow
+  license: Apache 2.0
+  summary: R Integration to 'Apache' 'Arrow'.
+  license_family: APACHE
+
+extra:
+  recipe-maintainers:
+    - conda-forge/r
```

Comment:
Feel free to add @conda-forge/arrow-cpp as a maintainer, we can then update this package with all other Arrow packages when we make a release.

Reply:
ok. done